filters/log/log.go

```diff
@@ -157,3 +165,52 @@ func (al *auditLog) Response(ctx filters.FilterContext) {
 		log.Errorf("Failed to json encode auditDoc: %v", err)
 	}
 }
+
+type unverifiedAuditLog struct {
+	writer io.Writer
+}
+
+// NewUnverifiedAuditLog logs "Sub" of the middle part of a JWT Token.
+func NewUnverifiedAuditLog() filters.Spec { return &unverifiedAuditLog{writer: os.Stderr} }
```

Comment:
it would be nicer if the writer was a parameter, and the caller side could decide what to pass in

Reply:
dropped writer